flow360/component/simulation/operating_condition.py

```diff
@@ -112,6 +112,11 @@ def dynamic_viscosity(self) -> ViscosityType.Positive:
         # return self.material.speed_of_sound(self.temperature)
         return 1.825e-5 * u.Pa * u.s
 
+    @pd.validate_call
+    def mu_ref(self, mesh_unit: LengthType.Positive) -> pd.PositiveFloat:
+        """Computes nondimensional dynamic viscosity."""
+        return (self.dynamic_viscosity / (self.speed_of_sound * self.density * mesh_unit)).v.item()
```

Comment:
I would not implement this logic here. Unit conversions should be independent from data models. I would prefer we do:
mu_ref = dynamic_viscosity.in_base(unit_system='flow360")
or this kind of interface. 

basically `mu_ref` is of the same type like `dynamic_viscosity` with only difference in unit. Thus I would not create special case and arithmetic for it here. See how it was done before: 
https://github.com/flexcompute/Flow360/blob/4821053f8a1f570f4d0b5884042f73c9e8936efa/flow360/component/flow360_params/flow360_params.py#L643

This `ExtraDimensionedProperty` was introduced for values that di not exist originally in the model. We might not have this issue in operating conditions.

Reply:
The difficulty is really how I access this nondimensional viscosity in the converter. muRef used to be part of what we store in Freestream for flow360_params but that's not the case here. We consider making muRef a private attribute, but that requires special treatment for serialization and deserialization. I'm not sure if it's worth the effort

Reply:
@yifan-flex muRef I think is not part of user interface. We use `ExtraDimensionedProperty` to compute it actually. Similar to what you did here.

Reply:
But a Freestream object is returned in to_solver? How do I do something similar?

Reply:
It is possible because Flow360Param accept other models that has mu_ref. I do think we should move away from letting user compute the nondimensional parameter.

Reply:
You can try something like: https://github.com/flexcompute/Flow360/blob/a95b0d442eb477e72d8f1763a1aba49794279e52/tests/simulation/translator/test_surface_meshing_translator.py#L42C23-L42C24

Reply:
Adding TODO for using unit system for nondimensionalization